webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -46,17 +72,19 @@ def response_correlation_controller(parent: WebvizErtPluginABC, app: dash.Dash)
             State(parent.uuid("parameter-selection-store-param"), "data"),
             State(parent.uuid("parameter-selection-store-resp"), "data"),
             State(parent.uuid("selected-ensemble-dropdown"), "value"),
+            State(parent.uuid("correlation-store-selected-obs"), "data"),
```

Comment:
Why is the component property of `correlation-store-selected-obs` different in Input (as modified_timestamp) and State (as data)?

Reply:
This is a good question, there is  another places in the code where this pattern is used:

here:
```
    @app.callback(
        Output(
            parent.uuid("response-overview"),
            "figure",
        ),
        [
            Input(parent.uuid("correlation-store-xindex"), "modified_timestamp"),
            Input(parent.uuid("parameter-selection-store-resp"), "modified_timestamp"),
            Input(parent.uuid("ensemble-selection-store"), "modified_timestamp"),
            Input(parent.uuid("correlation-store-selection"), "modified_timestamp"),
            Input(parent.uuid("correlation-store-selected-obs"), "modified_timestamp"),
        ],
        [
            State(parent.uuid("parameter-selection-store-resp"), "data"),
            State(parent.uuid("ensemble-selection-store"), "data"),
            State(parent.uuid("correlation-store-xindex"), "data"),
            State(parent.uuid("correlation-store-selection"), "data"),
            State(parent.uuid("correlation-store-selected-obs"), "data"),
        ],
    )
    def update_response_overview_plot(
```

And I kind of assumed it was required because there is an issue with triggering the callback if we only had for example `Input(parent.uuid("correlation-store-selected-obs"), "data")` and we needed  `Input(parent.uuid("correlation-store-selected-obs"), "modified_timestamp")` to trigger the callback for every change in the state of parent.uuid("correlation-store-selected-obs"), but I never checked if that is true or not. 

I give it a test and see if I can eliminate the state and only use the input.

Reply:
gave it a quick test and it seems there is no obvious advantage to using the pattern:
```python
    Input(parent.uuid("correlation-store-selected-obs"), "modified_timestamp"),
    State(parent.uuid("parameter-selection-store-param"), "data"),
 ```
I just replaced it with 
```python
    Input(parent.uuid("correlation-store-selected-obs"), "data"),
```

Seems to work the same